test/e2e/resources/components/localization/source/main.brs

```diff
@@ -0,0 +1,10 @@
+sub main()
+    _brs_.process.setLocale("en_US")
+    print tr("Hello") ' => Bonjour
+    print tr("hello") ' => hello
```

Comment:
is the `tr` function case sensitive? I haven't had to use it yet 😅

Reply:
No functions are case-sensitive in RBI! 😃

Reply:
Sorry I didn't ask this correctly, I was talking about the argument. Is `tr("Hello") == tr("hello")`? I assume that Roku doesn't use case insensitive for the translations in the XML file, right?

Reply:
Oh d'oh!  I actually don't know!  Great catch — @lkipke would you mind checking RBI's behavior here?

Reply:
Yep, I should have added a comment -- this is how RBI behaves 😄